### PR TITLE
Add convener/reviewer views

### PR DESCRIPTION
**Pull request by ThiefMaster**
Supersedes #2666

### CODE REVIEW

File: indico/modules/events/abstracts/templates/display/tracks.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                    <div class="track-review-row">
+                        <li>
+                            <div class="title">
+                                <a href="{{ url_for('abstracts.display_reviewable_track_abstracts', track) }}">
+                                    {{- track.title -}}
+                                </a>
+                                <span class="badge"
+                                      title="{{ abstract_count[track]['total'] }} {% trans %}total{% endtrans %}">
+                                    {{- abstract_count[track]['total'] - abstract_count[track]['reviewed'] }}
+                                    {% trans %}left to review{% endtrans -%}
```
Comment (pferreir) [inline indico/modules/events/abstracts/templates/display/tracks.html]: I think we should omit `0 left to review`. Or replace it with a green sticker saying "Done" or similar.

Comment (pferreir) [inline indico/modules/events/abstracts/templates/display/tracks.html]: ... especially when the total number of reviewable abstracts is also zero.

Comment (ThiefMaster) [inline indico/modules/events/abstracts/templates/display/tracks.html]: We should probably replace the "x total" with "x already reviewed" or change the logic. Right now the "total" is the number of abstracts that are being reviewed_for the track which are still in the 'submitted' state

File: indico/htdocs/sass/modules/abstracts/_abstract_list.scss [hunk 2]
```diff
@@ -18,3 +18,12 @@
 .submitter-column {
     text-align: center;
 }
+
+.accepted-on-other-track{
+    td {
+        color: #777;
+    }
+    a {
```
Comment (nop33) [inline indico/htdocs/sass/modules/abstracts/_abstract_list.scss]: There should be an empty line above.

File: indico/modules/events/abstracts/blueprint.py [hunk 3]
```diff
@@ -23,8 +23,9 @@
                                                                          RHEditEmailTemplateText, RHEmailTemplateList,
                                                                          RHDeleteEmailTemplate, RHSortEmailTemplates,
                                                                          RHEmailTemplateREST)
-from indico.modules.events.abstracts.controllers.management import (RHAbstracts, RHAbstractList,
-                                                                    RHAbstractListCustomize, RHAbstractListStaticURL,
+from indico.modules.events.abstracts.controllers.management import (RHAbstractList,
+                                                                    RHAbstracts, RHAbstractListCustomize,
+                                                                    RHAbstractListStaticURL,
```
Comment (nop33) [inline indico/modules/events/abstracts/blueprint.py]: These 3 lines can easily become 2.

Comment (ThiefMaster) [inline indico/modules/events/abstracts/blueprint.py]: And make rebase conflicts more likely. :/

I'd rather change the imports to just import modules here later once this and the other PRs are merged and then use e.g. `management.RHWhatever` in the rule definitions.

**Pull request merged**